docker-compose.yml

```diff
@@ -9,8 +9,8 @@ services:
     ports:
       - "9092:9092"
     environment:
-      KAFKA_ADVERTISED_HOST_NAME: 172.17.0.1
-      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+      KAFKA_ADVERTISED_HOST_NAME: ${DOCKER_HOST_IP}
+      KAFKA_ZOOKEEPER_CONNECT: ${DOCKER_HOST_IP}:2181
       KAFKA_CREATE_TOPICS: "topic.J2735BsmFilteredJson:1:1,topic.J2735BsmRawJson:1:1"
```

Comment:
Please change the default topic names to `topic.OdeBsmJson` and `topic.FilteredOdeBsmJson`. For TIM sanitization, the topic names would be `topic.OdeTimJson` and `topic.FilteredOdeTimJson`.